tests/functional/ProjectTask.php

```diff
@@ -402,39 +402,49 @@ public function testCloneProjectTask()
         $this->string($clonedSubtask2->fields['name'])->isEqualTo($subtask2->fields['name'] . ' (copy)');
     }
 
-    public function testAutoSetDateForAdd()
+    public function dataTestAdd()
```

Comment:
Some cases are not tested :
 - a project task created without a `percent_done` input
 - a project task created with an empty `percent_done` input
 - a project task created with empty strings in `real_start_date` and/or `real_end_date` input
 - a project task created with null values in `real_start_date` and/or `real_end_date` input